adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/android/PlatformParameters.cs

```diff
@@ -39,6 +39,9 @@ public class PlatformParameters : IPlatformParameters
     {
         internal CoreUIParent CoreUIParent { get; private set; }
 
+        /// <summary>
+        /// Constructor
```

Comment:
We are not going to remove it, but I question the value of this default constructor. No parent activity, does not set the broker and prompt behavior. I don't understand the scenarios?